interfaces/apparmor/apparmor.go

```diff
@@ -35,6 +35,17 @@ import (
 	"github.com/snapcore/snapd/osutil"
 )
 
+// ValidateFreeFromAARE will check that the given string does not
+// contain AppArmor regular expressions (AARE) or double quotes
+func ValidateFreeFromAARE(s string) error {
```

Comment:
I suggest _ValidateNoAppArmorRegexp_ instead, as something that won't require readers to come looking for the implementation to understand the code in the call site.